src/View/Helper/IdentityHelper.php

```diff
@@ -44,7 +55,7 @@ class IdentityHelper extends Helper
      */
     public function initialize(array $config): void
     {
-        $this->_identity = $this->getView()->getRequest()->getAttribute('identity');
+        $this->_identity = $this->_View->getRequest()->getAttribute($this->getConfig('identityAttribute'));
```

Comment:
Is the `$config` parameter supposed to override default?

Reply:
Yes.

Reply:
Why do we pass $config to initialize() in helpers?

Reply:
I don't really remember why. But it does allow you to differentiate between options that were explicitly passed vs the config merged with defaults.